### PR TITLE
Fix the website Dor created and implement smoothing

### PR DESCRIPTION
**Pull request by orel-david**
general
Added some parameters to the list, changed the grid of the website and its look, fix some typos and solvable errors.
Note: in the JavaScript parts there is some errors which shouldn't be fixed.

options added: 
- if you want to adjust the smoothing function you need to adjust its parameters.
- if you want to take the json and transform to csv file save it as a key and click "save file", then take the file to the embedded website to the "upload json here".

### CODE REVIEW

File: path_generator/fillet.js [hunk 1]
```diff
@@ -64,6 +65,48 @@ function getRotationPoints(startPt, middlePt, endPt, r, step) {
...
+            if (getDist(startPt,endAngle)> parameters.weightData) rotationPoints.push(getPointByAngle(angle, middlePt, isRight, r, a, b));
+        }
+    } else {
+        for (var angle = startAngle; angle < endAngle; angle += step) {
+            rotationPoints.push(getPointByAngle(angle, middlePt, isRight, r, a, b));
+        }
+    }
+    rotationPoints.push(getPointByAngle(endAngle, middlePt, isRight, r, a, b));
+    return rotationPoints;
+   /* var newPath = parameters.pathPoints.slice(0,parameters.pathPoints.length);
```
Comment (Repulser) [inline path_generator/fillet.js]: Remove or use commented out code

File: path_generator/fillet.js [hunk 2]
```diff
@@ -75,4 +118,12 @@ function getRotationPoints(startPt, middlePt, endPt, r, step) {
     }
     rotationPoints.push(getPointByAngle(endAngle, middlePt, isRight, r, a, b));
     return rotationPoints;
-}
\ No newline at end of file
+   /* var newPath = paramet*/
+function smoother() {
```
Comment (Repulser) [inline path_generator/fillet.js]: Unused method

File: path_generator/index.html [hunk 3]
```diff
@@ -302,9 +337,48 @@ <h4>Code Preview (copy & paste this code into a CommandGroup)</h4>
 // Make sure the image is loaded first otherwise nothing will draw.
         background.onload = drawField;
         $(canvas).mousemove(function (e) {
-            let rect = this.getBoundingClientRect(),
-                x = e.clientX - rect.left,
-                y = e.clientY - rect.top;
+            $(this).toggleClass('active');
+            var rect = this.getBoundingClientRect(),
+                x = e.clientX - rect.left, y = e.clientY - rect.top;
+            /*          $(canvas).onkeypress(function (z) {
```
Comment (Repulser) [inline path_generator/index.html]: Remove or use commented out code

**Comment by DanKatzuv** [comment]
Good work!

**Pull request merged**